openshift/templates/keycloak.yaml

```
@@ -198,10 +198,11 @@ parameters:
 - name: DM_APP_ROOT_URL
   displayName: Digital Marketplace Root Url
   description: Digital Markerplace Application root Url
-  value: http://marketplace-project-name.apps.exp.lab.pocquebec.org
+  #value: http://marketplace-project-name.apps.exp.lab.pocquebec.org
+  value: https://keycloak-digital-marketplace.apps.exp.openshift.cqen.ca
   required: true
 - name: KEYCLOAK_URL
   displayName: Keycloak Url
   description: Planned Keycloak Url
-  value: http://keycloak-project-name.apps.exp.lab.pocquebec.org
+  value: https://keycloak-digital-marketplace.apps.exp.openshift.cqen.ca
```

```suggestion
  value: https://keycloak-project-name.apps.exp.openshift.cqen.ca
```
Il serait préférable de ne pas spécifier le nom du projet et de le laisser générique afin que le développeur puisse le modifier en fonction de son projet.

---

https://github.com/CQEN-QDCE/digital_marketplace/commit/d9eac02dbee1a71cbda60923cf45a9502b52ea26

---

CLUSTER_DOMAIN, POD_NAMESPACE, elles ne sont pas déclarées dans cette template, ce n'est pas la même chose que marketplace.yaml
Il faudrait trouver un moyen de les déclarer de la même manière ou de les mettre en code dur, et je préfère la première option, je ne sais pas ce que tu en penses ?